lesson5.js

```diff
@@ -0,0 +1,30 @@
+  const ul = document.getElementById('js-list');
+  const info = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+    ];
+  const fragment = document.createDocumentFragment();
+
+  function list() {
+    return new Promise((resolve) => {
+      resolve(info);
+    });
+  }
+  list().then(() => { 
```

Comment:
Now, let's try to receive the resolved value with `then()`.
Currently, we are getting the value directly from info.

 [This part](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then#return_value) of MDN is easy to understand.

Reply:
Thank you for providing MDN information. I tried to fix it, but I'm not confident, so I would appreciate it if you could tell me.

I've fixed it a little, but I'd be happy if you could check if the direction is right.

Reply:
My explanation was inadequate.
I apologize for making you use your valuable time.

I wrote an example in [CodeSandBox](https://codesandbox.io/s/optimistic-platform-vby8t?file=/index.js).
The `hoge` part of the argument name is not appropriate and should be changed.

The value passed as an argument to resolve() can be received then().

```
  function makeLinkList() {
    return new Promise((resolve,reject) => {
      resolve(info);
    });
  }
makeLinkList().then((hoge) => {
  ListContents(hoge);
});

```
I suggest you do `console.log(hoge) `to check it out.


If you have any questions, please ask.

Reply:
I am always grateful for your help. I understand very much.